src/Layout-test-utils.js

```diff
@@ -337,11 +371,13 @@ var layoutTestUtils = (function() {
     div.style.display = 'flex';
     div.style.flexDirection = 'column';
     div.style.alignItems = 'flex-start';
+    div.style.alignContent = 'stretch';
```

Comment:
I found flexbox defaults to be pretty terrible, the ones that are in React native work really well to build apps. Also, when I import this back to fb codebase, I don't want everything to break under me!